.travis.yml

```diff
@@ -80,10 +93,12 @@ install:
   - if [ "${JOB_NAME}" == cmake-mingw ]; then
       sudo apt-get install -y mingw-w64 ;
     fi
-  - if [[ "${JOB_NAME}" == cmake* ]] && [ "${TRAVIS_OS_NAME}" == linux ]; then
+  - if [[ "${JOB_NAME}" == cmake* ]] && [ "${TRAVIS_OS_NAME}" == linux ] && [ "$(uname -m)" == x86_64 ]; then
       mkdir cmake-dist && curl --silent --fail --show-error --location https://github.com/Kitware/CMake/releases/download/v3.14.5/cmake-3.14.5-Linux-x86_64.tar.gz | tar --strip-components=1 -C cmake-dist -xz && export PATH=$PWD/cmake-dist/bin:$PATH;
+    elif [[ "${JOB_NAME}" == cmake* ]] && [ "${TRAVIS_OS_NAME}" == linux ] && [ "$(uname -m)" == ppc64le ]; then
+      mkdir cmake-dist && curl --silent --fail --show-error --location https://github.com/Kitware/CMake/releases/download/v3.14.5/cmake-3.14.5.tar.gz | tar --strip-components=1 -C cmake-dist -xz && mkdir cmake-dist/build && pushd cmake-dist/build && ../bootstrap && make && popd && export PATH=$PWD/cmake-dist/build/bin:$PATH;
```

Comment:
Building from source looks to take almost 20 minutes. Can you put together a binary .tar.bz2 for this?

Reply:
I already asked upstream, but no progress - https://gitlab.kitware.com/cmake/cmake/issues/20122

I could build and host our own?

Reply:
@pdillinger here ya go, binaries for ppc64le - http://static.adamretter.org.uk/cmake-3.14.5-Linux-ppc64le.tar.bz2

Reply:
Hmm. Is that build missing a download capability? https://travis-ci.org/facebook/rocksdb/jobs/660214094?utm_medium=notification&utm_source=github_status

Reply:
Ah! It looks like the cmake I build for ppc64le maybe doesn't support HTTPS (SSL) - let me build a new one...

Reply:
@pdillinger I have uploaded a newer [cmake-3.14.5-Linux-ppc64le.tar.bz2]( http://static.adamretter.org.uk/cmake-3.14.5-Linux-ppc64le.tar.bz2), hopefully this one includes SSL support. The new versions md5 checksum is `5dd300852435fa9eca7d519098bed2a0`. 

Can you replace your copy and re-trigger the build?

Reply:
Done